gmso/core/parametric_potential.py

```diff
@@ -15,36 +19,26 @@ class Potential(object):
     potential, which can be e.g. a non-bonded potential, a bonded potential, an
     angle potential, a dihedral potential, etc. and is designed to be inherited
     by classes that represent these potentials.
+    """
 
-    Parameters
-    ----------
-    name : str, default="Potential"
-        The name of the potential.
-    expression : str or sympy.Expr, default='a*x+b'
-        The mathematical expression describing the functional form of the
-        potential.
-    parameters : dict {str: unyt.unyt_quantity},
-            default={'a': 1.0*u.dimensionless, 'b': 1.0*u.dimensionless}
-        The parameters of the potential and their values, as unyt quantities.
-        The keys are names of the variables included in `expression` and values
-        are the numerical values of these parameters recorded as instances of
-        `unyt.unyt_quantity`, which combine both value and unit information.
-    independent_variables : str or sympy.Symbol or list or set thereof
-        The independent variables in the expression of the potential.
-    topology: gmso.core.Topology, the topology of which this potential is a part of, default=None
-    set_ref: (str), the string name of the bookkeeping set in topology class.
+    topology_: Optional[Any] = Field(
```

Comment:
`Optional[Any]` should be gmso.topology

Reply:
Fixed with validator. Delegating this for later issue/pr.